modules/machineset-yaml-osp.adoc

```diff
@@ -78,3 +79,4 @@ $ oc get -o jsonpath='{.status.infrastructureName}{"\n"}' infrastructure cluster
 ----
 <2> Specify the node label to add.
 <3> Specify the infrastructure ID and node label.
+<4> Enter the value that is returned when link:https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/16.0/html/command_line_interface_reference/server#server_group_create[creating a server group]. The server group must have `anti-affinity` or `soft-anti-affinity` policies.
```

Comment:
Actually, while I can't imagine a reason for setting `affinity` or `soft-affinity`, any policy (or none) is fine as far as OpenShift is concerned.

Reply:
I'd go like:
"Can be used to set an anti-affinity policy for the MachineSet".